internal/service/project/model_project.go

```diff
@@ -202,3 +206,32 @@ func UpdateProjectBool(plan, state types.Bool, setting **bool) bool {
 	}
 	return false
 }
+
+func NewTFTags(tags *[]admin.ResourceTag) types.Map {
+	if tags == nil {
+		return types.MapNull(types.StringType)
+	}
+
+	typesTags := make(map[string]attr.Value, len(*tags))
+	for _, tag := range *tags {
+		typesTags[tag.Key] = types.StringValue(tag.Value)
+	}
+	return types.MapValueMust(types.StringType, typesTags)
+}
+
+func NewAdminTags(ctx context.Context, tags types.Map) *[]admin.ResourceTag {
```

Comment:
```suggestion
func NewResourceTags(ctx context.Context, tags types.Map) *[]admin.ResourceTag {
```

Reply:
fixed in 35d729e4e186219cab55b274cb1af334b82e1645